add-ons/plugin.video.tvvlaanderen
# Improve tests

Tests:
* Replace `coverage` with `pytest-cov`
* Remove `tox`
* Add tests for IPTV Manager and Library
* Fix token verification with pyjwt 2.0

Github Actions:
* Don't run actions twice (push & pull_request), see https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662
* Use `xbmc/action-kodi-addon-checker@v1.2` action instead of installing ourselves
* Use output of `xbmc/action-kodi-addon-submitter` for the Github release
